db.go

```diff
@@ -13,7 +13,7 @@ type Migratable interface {
 type Postgres struct{}
 
 func (p Postgres) SelectMigrationTableSql() string {
-	return "SELECT tablename FROM pg_catalog.pg_tables WHERE tablename = $1"
+	return "SELECT table_name FROM information_schema.tables WHERE table_name= $1 AND table_catalog = $2"
```

Comment:
No sir; the intent is to isolate/restrict the migration to only the database to which we're connected.  It appears that PG will likely enforce this by default, whereas MySQL doesn't.  The checks added to PG in this PR may not be required, but also do not hurt either as they enforce this explicitly (rather than implicitly through the connection).